src/app/_data/cmsConfigSchema.json

```diff
@@ -24,6 +24,27 @@
       }
     ]
   },
+  "image_config": {
+    "name": "image",
+    "label": "Image",
+    "widget": "object",
+    "fields": [
+      {
+        "name": "url",
+        "label": "URL",
+        "widget": "image",
+        "allow_multiple": false,
+        "hint": "Use https://squoosh.app/ to compress images before uploading. Choose the WebP format and resize the image to 2000px width."
```

Comment:
Should this be 200px width?

Reply:
No – The largest size is `1000px` - 2x for Retina to be on the safe side.